docs/conceptual/array.sumby['t,^u]-function-[fsharp].md

```diff
@@ -30,29 +30,41 @@ Array.sumBy projection array
 ```
 
 #### Parameters
-*projection*
-Type: **'T -&gt; ^U**
-
+*projection* Type: **'T -&gt; ^U**
 
 The function to transform the array elements into the type to be summed.
 
-
-*array*
-Type: **'T**[[]](https://msdn.microsoft.com/library/def20292-9aae-4596-9275-b94e594f8493)
+*array* Type: **'T**[[]](https://msdn.microsoft.com/library/def20292-9aae-4596-9275-b94e594f8493)
 
 
 The input array.
 
 
+## Return Value
+The sum of the results of applying the projection function to each element of the array.
+
+
+## Exceptions
+May throw an [OverflowException](https://msdn.microsoft.com/library/system.overflowexception.aspx) due to arithmetic overflows.
+
 
-**The resulting sum.**
 ## Remarks
+The results are summed using the **[checked + operator](https://msdn.microsoft.com/visualfsharpdocs/conceptual/checked.[-p-][%5Et1%2c%5Et2%2c%5Et3]-function-[fsharp])** and **Zero** property associated with the summed type.
+
 This function is named **SumBy** in compiled assemblies. If you are accessing the function from a language other than F#, or through reflection, use this name.
 
-**The following code shows how to use Array.sumBy.**
```

Comment:
I agree here that this particular text is not needed.  If it were different and explained the sample then I'd argue for it to still exist, but as-is it doesn't state anything useful.